letsdrill.rb

```diff
@@ -1,12 +1,23 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
-
+  if (90..100).include?(integer)
+  	"A"
+  elsif (80..89).include?(integer)
+  	"B"
+  elsif (70..79).include?(integer)
+  	"C"
+  elsif (60..69).include?(integer)
+  	"D"
+  else 
+  	"F"
+  end
 end
 
 def shortest_string(array)
 
-  #Put your code here!
+  return nil if array.empty?
```

Comment:
I don't think this line is necessary as `min_by` will return `nil` by default if the array is empty.  Nice use of the `&` method syntax though 😄